Tests/AttributedStringStyleTests.swift

```diff
@@ -555,6 +555,134 @@ class StringStyleTests: XCTestCase {
         XCTAssertFalse(stillHasAltSixDict)
     }
 
+    func testStyleBlockRules() {
+        let string = "0<one attr1=\"11\">1<two attr2=\"12\">2</two></one>"
+
+        var tagAttr1Value: String!
```

Comment:
No need for these to be `!`, since `XCTAssertEqual` does optional-promotion for you. i.e. this can be `String?`, and it will Do The Right Thing™